extra/material.js

```diff
@@ -2457,12 +2457,6 @@ function MaterialTab(tab, ctx) {
         }
         tab.addEventListener('click', function (e) {
             e.preventDefault();
-            var href = tab.href.split('#')[1];
-            var panel = ctx.element_.querySelector('#' + href);
-            ctx.resetTabState_();
-            ctx.resetPanelState_();
-            tab.classList.add(ctx.CssClasses_.ACTIVE_CLASS);
-            panel.classList.add(ctx.CssClasses_.ACTIVE_CLASS);
```

Comment:
The problem here is that we try to avoid making modifications to the base MDL library. And the code you're removing here is the reason why the Tab(s) component doesn't implement the `upgrade` function.

It would be better to find another solution to resolve the issue without making any modifications to the parent library. If this is not possible, it would be interesting to ask on the MDL repository if we can find a better way to handle this case. Maybe just add a check on the href before trying to reset the panels should be enough and maybe they will accept this small change.